src/Symfony/Component/String/composer.json

```diff
@@ -24,12 +24,13 @@
     },
     "require-dev": {
         "symfony/error-handler": "^5.4|^6.0",
-        "symfony/intl": "^6.2",
+        "symfony/emoji-transliterator": "^6.3",
```

Comment:
maybe put the `emoji-transliterator` in composer's `suggest` section and explain its use

Reply:
We stopped using the suggest section IIRC

Reply:
Indeed